fitwsarah_frontend/src/LanguageConfig/i18n.js

```diff
@@ -0,0 +1,24 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from "i18next-browser-languagedetector";
+import Backend from 'i18next-http-backend';
+
+i18n
+.use(LanguageDetector)
+.use(initReactI18next)
+.use(Backend)
+.init({
+    debug: false,
+    fallbackLng: 'en',
+    interpolation: {
+        escapeValue: false,
+    },
+    backend: {
+        loadPath: '/LanguageContext/{{lng}}/{{ns}}.json',
```

Comment:
Could you elaborate on how loading this .json file makes the localization work?

Reply:
I will explain in a call its easy but will be easier if I show you